### PR TITLE
footer part has been designed

### PR DESCRIPTION
**Pull request by SomaskaranParamanantham**

### CODE REVIEW

**Review by UmeshikaMahaliyadda — APPROVED** [review]
if you can move social media platforms into right side corner, it will look nice according to my view. Other than that, nice code.

File: pregred-gui/src/app/components/app-footer/AppFooter.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import './AppFooter.css';
+import {
+  FacebookOutlined,
+  InstagramOutlined,
+  YoutubeOutlined,
+  TwitterOutlined
+} from '@ant-design/icons';
+
+
```
Comment (Nisheanthan) [inline pregred-gui/src/app/components/app-footer/AppFooter.js:10]: only leave one line space between imports and function

File: pregred-gui/src/app/components/app-footer/AppFooter.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+
+function AppFooter() {
+    return(
+      <div className="footer-container">
+        <div className="f-text">
```
Comment (Nisheanthan) [inline pregred-gui/src/app/components/app-footer/AppFooter.js:16]: be more specific on CSS class names. 'dot' and 'f-text' are not understandable. if it is footer text, name that as footer-text.

File: pregred-gui/src/app/components/app-footer/AppFooter.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+          <ul className="dot">
+            <li>About Us: <a href="www.pregred.com"> www.pregred.com </a> </li>
+            <li>Contact Us: 0768660203</li>
+          </ul>
+          <div className="social">
+            <p >Social  follows</p>
```
Comment (Nisheanthan) [inline pregred-gui/src/app/components/app-footer/AppFooter.js:22]: Social follows - what does this mean? focus on whether the content makes sense. 
moreover, are we going to link facebook, insta, linked in profiles here? if not remove them.

File: pregred-gui/src/app/components/app-footer/AppFooter.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+
+
+function AppFooter() {
+    return(
+      <div className="footer-container">
+        <div className="f-text">
+          <ul className="dot">
+            <li>About Us: <a href="www.pregred.com"> www.pregred.com </a> </li>
+            <li>Contact Us: 0768660203</li>
```
Comment (Nisheanthan) [inline pregred-gui/src/app/components/app-footer/AppFooter.js:19]: As i know, About Us and Contact Us  are going to navigate to respective pages. so, 'www.pregred.com' and mobile numbers are not needed.

File: pregred-gui/src/app/components/app-footer/AppFooter.css [hunk 5]
```diff
@@ -0,0 +1,29 @@
+.footer-container {
+  bottom: 0;
+  width: 100%;
+  background-color: #000000;
```
Comment (Nisheanthan) [inline pregred-gui/src/app/components/app-footer/AppFooter.css:4]: Can you make this bg colour similar to the navbar background colour?. its a variety of dark blue.

**Review by Nisheanthan — CHANGES_REQUESTED** [review]
Changes to be done

**Pull request merged**